app/(app)/feedback.tsx

```diff
@@ -30,6 +32,16 @@ const Feedback = () => {
     setShowModal(true);
   };
 
+  const { mutate } = useMutation({
+    mutationFn: () => sendFeedback(description, selectedRating),
+    onSuccess: () => {
+      openModal();
+    },
+    onError: (error) => {
+      console.error(error);
```

Comment:
We should make a TODO here to implement a proper UI to inform the user that an error occurred.